xapian-applications/omega/query.cc

```diff
@@ -1228,6 +1233,10 @@ get_subdbs()
     return subdbs;
 }
 
+// mersenne twister for RNG
+static mt19937 rng;
+static bool seed_set = 0;
```

Comment:
It's clearer to use `false` and `true` as boolean literals rather than `0` and `1` (here and below).

I only noticed after merging, so I've just fixed this in a follow-up commit.